src/libraries/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.cs

```diff
@@ -41,5 +42,26 @@ public static string FrameworkDescription
                 return s_frameworkDescription;
             }
         }
+
+        /// <summary>
+        /// Returns an opaque string that identifies the platform on which an app is running.
+        /// </summary>
+        /// <remarks>
+        /// The property returns a string that identifies the operating system, typically including version,
```

Comment:
Add a link to https://docs.microsoft.com/en-us/dotnet/core/rid-catalog for more details?